app/_src/guides/transition-to-the-new-policies.md

```diff
@@ -0,0 +1,411 @@
+    * `route/idleTimeout` is duplicated value of `streamIdleTimeout` but per-route. Previously we've set it only per-listener.
+
+    These 3 facts perfectly explain the list of changes we're observing.
+
+3. Remove the `kuma.io/effect: shadow` label.
+   Although the old Timeout and the new MeshTimeout currently coexist, the new policy completely overrides the old one.
+
+4. Observe the demo app behaves as expected. If everything goes well, we can safely remove Timeout and conclude the transition.
+
+### CircuitBreaker -> MeshCircuitBreaker
+
+1. Create a replacement policy for `cb-global` CircutBreaker and apply it with `kuma.io/effect: shadow` label:
+
+    ```sh
+    echo 'apiVersion: kuma.io/v1alpha1
+    kind: MeshCircuitBreaker
+    metadata:
+      namespace: {{site.mesh_namespace}}
+      name: cb-global
+      labels:
+        kuma.io/mesh: default
+        kuma.io/effect: shadow
+    spec:
+      targetRef:
+        kind: Mesh
+      to:
+      - targetRef:
+          kind: Mesh
+        default:
+          connectionLimits:
+            maxConnections: 24
+            maxPendingRequests: 25
+            maxRequests: 26
+            maxRetries: 27
+          outlierDetection:
+            interval: 21s
+            baseEjectionTime: 22s
+            maxEjectionPercent: 23
+            splitExternalAndLocalErrors: false
+            detectors:
+              totalFailures:
+                consecutive: 28
+              gatewayFailures:
+                consecutive: 29
+              localOriginFailures:
+                consecutive: 30
+              successRate:
+                requestVolume: 31
+                minimumHosts: 32
+                standardDeviationFactor: "1.33"
+              failurePercentage:
+                requestVolume: 34
+                minimumHosts: 35
+                threshold: 36' | kubectl apply -f-
+    ```
+
+2. Check the list of changes for `redis_kuma-demo_svc_6379` pod in Envoy configuration using `kumactl`, `jq` and `jd`:
+
+    ```sh
+    kumactl inspect dataplane demo-app-b4f98898-zxrqj.kuma-demo --type=config --shadow --include=diff | jq '.diff' | jd -t patch2jd
+    ```
+   
+    As we can see this time is empty. CircuitBreaker and MeshCircuitBreaker configures Envoy in the exact similar way.
+
+3. Remove the `kuma.io/effect: shadow` label.
+   Although the old CircuitBreaker and the new MeshCircuitBreaker currently coexist, the new policy completely overrides the old one.
+
+4. Observe the demo app behaves as expected. If everything goes well, we can safely remove CircuitBreaker and conclude the transition.
+
+### TrafficRoute -> MeshTCPRoute
+
+It's safe to simply remove `route-all-default` TrafficRoute. 
+Traffic is going to flow through the system even if there are no either TrafficRoutes or MeshTCPRoutes/MeshHTTPRoutes.
+Ensure that TrafficRoute is the last policy deleted when removing older policies. 
```

Comment:
can we make this bold? or as some warn, to make sure people see it? This seems important

Reply:
added warning block